porcupine/plugins/filemanager.py

```diff
@@ -149,6 +154,98 @@ def update_dialog_state(*junk: object) -> None:
 
     return new_path
 
+def ask_directory_name(
```

Comment:
I don't like the copy/paste. Is there a reason why we can't just use `ask_file_name`?